web/src/beta/features/Editor/Tabs/Storytelling/StorySidePanel/index.tsx

```diff
@@ -0,0 +1,108 @@
+import { FC } from "react";
+import StorySidePanelAction from "src/beta/features/Editor/Tabs/Storytelling/StorySidePanelAction";
+import StorySidePanelItem from "src/beta/features/Editor/Tabs/Storytelling/StorySidePanelItem";
+import StorySidePanelPageWrapper from "src/beta/features/Editor/Tabs/Storytelling/StorySidePanelPageWrapper";
+
+import * as SidePanel from "@reearth/beta/features/Editor/SidePanel";
+import { styled } from "@reearth/services/theme";
+
+// TODO: these are currently rough definition
+type Props = {
+  stories: any;
+  selectedStory: any;
+  onSelectStory: (id: string) => void;
+  onStoryAdd: () => void;
+  selectedPageId?: string;
+  onSelectPage: (id: string) => void;
+  onPageAdd: () => void;
+};
+
+const StorySidePanel: FC<Props> = ({ onStoryAdd, onSelectStory, onPageAdd, onSelectPage }) => {
+  return (
+    <SidePanel.Wrapper location="left">
+      <SidePanel.Section maxHeight="33%">
+        <SidePanel.Card>
+          <SidePanel.Title>Story</SidePanel.Title>
```

Comment:
Please wrap text in i18n 🙇🏻 
Import useT from services/i18n and then do `const t = useT()` and then `t("Story")`

Reply:
Thank you fixed :pray: https://github.com/reearth/reearth/pull/565/commits/7177f8a5056938ff1e0b0fa2add09904ae883efa